### ISSUE TITLE
Current version of Ubuntu contains a number of HIGH security issues

### ISSUE THREAD

**Issue by shaqb** [sitespeedio/sitespeed.io#3603]
### Have you read the documentation?

- [X] Yes, I've read the [how to make a reproducable bug guide](https://www.sitespeed.io/documentation/sitespeed.io/bug-report/)
- [X] Yes, I've read the [how to debug my script guide](https://www.sitespeed.io/documentation/sitespeed.io/scripting/#debug)

### URL

n/a

### What are you trying to accomplish

When running a trivy scan on the 23.5.0 container, I can see there are a number of Ubuntu packages that have HIGH security issues against them.

```
+----------------+------------------+----------+--------------------+--------------------+---------------------------------------+
|    LIBRARY     | VULNERABILITY ID | SEVERITY | INSTALLED VERSION  |   FIXED VERSION    |                 TITLE                 |
+----------------+------------------+----------+--------------------+--------------------+---------------------------------------+
| libexpat1      | CVE-2022-25235   | HIGH     | 2.2.9-1build1      | 2.2.9-1ubuntu0.2   | expat: Malformed 2- and               |
|                |                  |          |                    |                    | 3-byte UTF-8 sequences can            |
|                |                  |          |                    |                    | lead to arbitrary code...             |
|                |                  |          |                    |                    | -->avd.aquasec.com/nvd/cve-2022-25235 |
+                +------------------+          +                    +                    +---------------------------------------+
|                | CVE-2022-25236   |          |                    |                    | expat: Namespace-separator characters |
|                |                  |          |                    |                    | in "xmlns[:prefix]" attribute         |
|                |                  |          |                    |                    | values can lead to arbitrary code...  |
|                |                  |          |                    |                    | -->avd.aquasec.com/nvd/cve-2022-25236 |
+----------------+------------------+          +                    +                    +---------------------------------------+
| libexpat1-dev  | CVE-2022-25235   |          |                    |                    | expat: Malformed 2- and               |
|                |                  |          |                    |                    | 3-byte UTF-8 sequences can            |
|                |                  |          |                    |                    | lead to arbitrary code...             |
|                |                  |          |                    |                    | -->avd.aquasec.com/nvd/cve-2022-25235 |
+                +------------------+          +                    +                    +---------------------------------------+
|                | CVE-2022-25236   |          |                    |                    | expat: Namespace-separator characters |
|                |                  |          |                    |                    | in "xmlns[:prefix]" attribute         |
|                |                  |          |                    |                    | values can lead to arbitrary code...  |
|                |                  |          |                    |                    | -->avd.aquasec.com/nvd/cve-2022-25236 |
+----------------+------------------+          +--------------------+--------------------+---------------------------------------+
| libssl1.1      | CVE-2022-0778    |          | 1.1.1f-1ubuntu2.10 | 1.1.1f-1ubuntu2.12 | openssl: Infinite loop in             |
|                |                  |          |                    |                    | BN_mod_sqrt() reachable               |
|                |                  |          |                    |                    | when parsing certificates             |
|                |                  |          |                    |                    | -->avd.aquasec.com/nvd/cve-2022-0778  |
+----------------+------------------+          +--------------------+--------------------+---------------------------------------+
| linux-libc-dev | CVE-2022-0001    |          | 5.4.0-97.110       | 5.4.0-104.118      | hw: cpu: intel: Branch                |
|                |                  |          |                    |                    | History Injection (BHI)               |
|                |                  |          |                    |                    | -->avd.aquasec.com/nvd/cve-2022-0001  |
+                +------------------+          +                    +                    +---------------------------------------+
|                | CVE-2022-0002    |          |                    |                    | hw: cpu: intel: Intra-Mode BTI        |
|                |                  |          |                    |                    | -->avd.aquasec.com/nvd/cve-2022-0002  |
+                +------------------+          +                    +--------------------+---------------------------------------+
|                | CVE-2022-0492    |          |                    | 5.4.0-105.119      | kernel: cgroups v1                    |
|                |                  |          |                    |                    | release_agent feature may             |
|                |                  |          |                    |                    | allow privilege escalation            |
|                |                  |          |                    |                    | -->avd.aquasec.com/nvd/cve-2022-0492  |
+                +------------------+          +                    +--------------------+---------------------------------------+
|                | CVE-2022-23960   |          |                    | 5.4.0-104.118      | hw: cpu: arm64: Spectre-BHB           |
|                |                  |          |                    |                    | -->avd.aquasec.com/nvd/cve-2022-23960 |
+                +------------------+          +                    +                    +---------------------------------------+
|                | CVE-2022-25636   |          |                    |                    | kernel: heap out of bounds            |
|                |                  |          |                    |                    | write in nf_dup_netdev.c              |
|                |                  |          |                    |                    | -->avd.aquasec.com/nvd/cve-2022-25636 |
+----------------+------------------+          +--------------------+--------------------+---------------------------------------+
| openssl        | CVE-2022-0778    |          | 1.1.1f-1ubuntu2.11 | 1.1.1f-1ubuntu2.12 | openssl: Infinite loop in             |
|                |                  |          |                    |                    | BN_mod_sqrt() reachable               |
|                |                  |          |                    |                    | when parsing certificates             |
|                |                  |          |                    |                    | -->avd.aquasec.com/nvd/cve-2022-0778  |
+----------------+------------------+----------+--------------------+--------------------+---------------------------------------+
```

Looking at the current version of Ubuntu focal, it looks like these are resolved in 20220316. Is it possible to update to this version to address the above security issues?

### What browser did you use?

Other

### How to reproduce

```shell
Run trivy scan on sitespeed.io 23.5.0 docker container
```

**Comment by shaqb** [comment]
> Thanks for creating the issue, I'll update when get some free time. We check for CRITICAL issues in the build process, for HIGH severities I think its best to make your own container, so you can be on top of those.

No worries @soulgalore, and yes that's a good shout on creating our own!

**Comment by soulgalore** [comment]
This should be fixed in 23.5.2

**Comment by shaqb** [comment]
You are amazing @soulgalore !
